components/sensor/pulse_meter.rst

```diff
@@ -6,8 +6,16 @@ Pulse Meter Sensor
     :image: pulse.svg
 
 The pulse meter sensor allows you to count the number and frequency of pulses on any pin. It is intended to be a drop-in replacement
-for :doc:`integration sensor </components/sensor/pulse_counter>`, but offering better resolution.
-It measures the time between rising edges on a pin, for each pulse it outputs the frequency in pulses/min.
+for the :doc:`pulse counter integration </components/sensor/pulse_counter>`.
+It measures the time between rising edges on a pin, for each pulse it outputs the frequency in ``pulses/min``, as opposed to the pulse counter sensor which outputs the number of pulses in a time interval.
+
+This leads to a higher resolution, especially for low pulse rates, as the pulse counter sensor is limited by the number of pulses within a time interval. The difference between ``1 pulse/min`` and ``2 pulses/min`` is much higher than ``2000 pulses/min`` and ``2001 pulses/min``. The pulse_meter is only limited by its accuracy to measure the time between pulses, which leads to an inverse relationship between pulse rate and accuracy. The lower the pulse rate, the higher the accuracy. Lower accuracy with very high pulse rates is less likely to be a problem, as the ESP32 can measure time intervals with a resolution of ``1us``.
```

Comment:
You've used back-ticks a lot to emphasise units - back-ticks are meant for code. I'm not sure it's even necessary to emphasise real-world units, but consider other options like ordinary quotes, italic or bold instead.

Just because ESP code can measure with a resolution of 1us does not mean it has an accuracy of 1us. I'd be inclined to remove that sentence.

Reply:
Mmh, the default was in backticks, I guess I misinterpreted it as a way to also mark units and such.

As to the resolution, the API talks about a microsecond resolution. https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/system/esp_timer.html